tests/test_condor_unit.py

```diff
@@ -145,3 +145,36 @@ def test_submit_dag_1(self, get_dag_file, needs_credentials):
             get_dag_file, TestUnit.test_vargs, TestUnit.test_schedd, cmd_args=[]
         )
         assert res
+
+
+class TestJob:
+    def test_job(self):
```

Comment:
Could you please mark all of these unit tests with the `@pytest.mark.unit` decorator?  We want to have a github action where all the unit tests are run for each PR, and that'll help us separate unit, system, and integration tests.

Reply:
OK